scripts/qa_crowbarsetup.sh

```diff
@@ -3253,6 +3253,9 @@ function get_ceph_nodes
 {
     if [[ $deployceph ]]; then
         cephmons=`crowbar ceph proposal show default | rubyjsonparse "puts j['deployment']['ceph']['elements']['ceph-mon']"`
+        for machine in $cephmons; do
+            cephmons_ips+="`crowbar machines show $machine | rubyjsonparse "puts j['crowbar']['network'][j['ceph']['client_network']]['address']"` "
```

Comment:
Can you adopt it to what we are doing in `get_node_name` function, please?
https://github.com/crowbar/crowbar-ceph/pull/84/files#diff-8aa7d82cf49f03965efd60be00f42157R9